builtin/logical/pki/path_issue_sign.go

```diff
@@ -192,21 +195,29 @@ func (b *backend) pathIssueSignCert(
 
 	case "pem_bundle":
 		resp.Data["issuing_ca"] = cb.IssuingCA
-		resp.Data["certificate"] = fmt.Sprintf("%s\n%s", cb.Certificate, cb.IssuingCA)
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
+		resp.Data["certificate"] = strings.TrimSpace(fmt.Sprintf("%s\n%s\n%s", cb.Certificate, cb.IssuingCA, cb.IssuingCAChain))
```

Comment:
Just to check, is the issuing CA not contained in the chain? If it is, probably want to remove appending `cb.IssuingCA` here.